lib/senders/sms.js

```diff
@@ -100,4 +101,32 @@ module.exports = function (log, translator, templates, smsConfig) {
       translator: translator.getTranslator(acceptLanguage)
     }).text
   }
+
+  function MockNexmo() {
+    return {
+      account: {
+        checkBalance (callback) {
+          log.info({ op: 'sms.balance.mock' })
+
+          callback(null, {
+            value: smsConfig.balanceThreshold
+          })
+        }
+      },
+      message: {
+        sendSms (senderId, phoneNumber, message, options, callback) {
```

Comment:
Unfortunately you can't use ES6 syntax in this directory at the moment, because the l10n parser doesn't understand it. But fwiw, I think this `MockNexmo` thing can just live in `lib`, `senders` is just a historical thing containing old mailer source, nothing more than that.